src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,18 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.view.LottoView;
+
+public class LottoController {
+	private static final LottoView view = new LottoView();
+
+	public static void main(String[] args) {
+		LottoCharge charge = LottoCharge.from(view.inputCharge());
+		Lottos lottos = Lottos.buy(charge);
+		view.showLottos(lottos);
+		Answer answer = new Answer(view.inputAnswer());
+		Winnings winnings = lottos.winnigs(answer);
+		Statistics statistics = Statistics.of(charge, winnings);
+		view.showStatistics(statistics);
```

Comment:
```suggestion
		LottoCharge charge = LottoCharge.from(view.inputCharge());
		Lottos lottos = Lottos.buy(charge);
		view.showLottos(lottos);
		
		Answer answer = new Answer(view.inputAnswer());
		Winnings winnings = lottos.winnigs(answer);
		
		Statistics statistics = Statistics.of(charge, winnings);
		view.showStatistics(statistics);
```
연관되는 비슷한 흐름끼리 묶어서 구분해주면 더 가독성이 좋을 것 같아요

Reply:
넵!👍🏻